README.md

```diff
@@ -237,3 +237,42 @@ def attacker_factory():
         }
     }
 ```
+
+### Allowlisting and denylisting Swagger tags and operations
```

Comment:
Does it make sense to keep the language consistent with what the decorators are called? i.e., `Including and excluding Swagger tags and operations`

Reply:
Makes sense, done!